src/writer_postgres.py

```
@@ -196,14 +210,12 @@
                 elif topicName == "public.cps.za.test":
                     postgres_test_write(cursor, "public_cps_za_test", message)
                 else:
-                    msg = f"unknown topic for postgres {topicName}"
-                    _logger.error(msg)
-                    return False, msg
-                    
+                    _logger.error(f"unknown topic for postgres {topicName}")
+                    return False
+
             connection.commit()
     except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)